src/numerical_flux/numerical_flux_factory.cpp

```diff
@@ -23,6 +25,10 @@ ::create_convective_numerical_flux(
         if constexpr (dim+2==nstate) return std::make_unique< L2Roe<dim, nstate, real> > (physics_input);
     } else if (conv_num_flux_type == AllParam::split_form) {
         return std::make_unique< SplitFormNumFlux<dim, nstate, real> > (physics_input);
+    } else if (conv_num_flux_type == AllParam::central_flux) {
+        return std::make_unique< CentralNumFlux<dim, nstate, real> > (physics_input);
+    } else if (conv_num_flux_type == AllParam::entropy_cons_flux) {
+        return std::make_unique< EntropyConsNumFlux<dim, nstate, real> > (physics_input);
```

Comment:
as @dougshidong mentioned, add `constexpr` for `dim==1` and `pde_type==PDE_eum::Burgers` (something like that) as a safeguard here

Reply:
fixed at parameters declaration level.

Reply:
which file / line?

Reply:
line 311 in src/parameters/all_parameters.cpp